<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>image_mode_widget</class>
 <widget class="QWidget" name="image_mode_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>110</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QTabWidget" name="tab_widget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="raw_tab">
      <attribute name="title">
       <string>Raw</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="raw_tabbed_view">
         <property name="text">
          <string>Tabbed View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="raw_show_saturation">
         <property name="text">
          <string>Show Saturation</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cartesian_tab">
      <attribute name="title">
       <string>Cartesian</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox{padding-top:15px; margin-top:-20px}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="cartesian_virtual_plane_distance">
              <property name="toolTip">
               <string>The distance from the source to the virtual plane.</string>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="value">
               <double>1000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="cartesian_pixel_size">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="cartesian_plane_normal_rotate_y">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Pixel Size:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QDoubleSpinBox" name="cartesian_plane_normal_rotate_x">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Rotate x:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Rotate y:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="toolTip">
               <string>The distance from the source to the virtual plane.</string>
              </property>
              <property name="text">
               <string>Distance:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>36</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="polar_tab">
      <attribute name="title">
       <string>Polar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox{padding-top:15px; margin-top:-20px}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="1" column="2">
               <widget class="QDoubleSpinBox" name="polar_res_tth_min">
                <property name="keyboardTracking">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>-</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>2θ:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QDoubleSpinBox" name="polar_pixel_size_tth">
                <property name="keyboardTracking">
                 <bool>false</bool>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>99.989999999999995</double>
                </property>
                <property name="singleStep">
                 <double>0.005000000000000</double>
                </property>
                <property name="value">
                 <double>0.050000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QDoubleSpinBox" name="polar_pixel_size_eta">
                <property name="keyboardTracking">
                 <bool>false</bool>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>99.989999999999995</double>
                </property>
                <property name="singleStep">
                 <double>0.005000000000000</double>
                </property>
                <property name="value">
                 <double>0.200000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>η:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Pixel Size:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>2θ Range:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QDoubleSpinBox" name="polar_res_tth_max">
                <property name="keyboardTracking">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>20.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="polar_apply_snip1d">
                <property name="text">
                 <string>Apply snip1d?</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QDoubleSpinBox" name="polar_snip1d_width">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string>°</string>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.200000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QSpinBox" name="polar_snip1d_numiter">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_10">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>w:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="label_11">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>n iter:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>2</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>raw_tabbed_view</tabstop>
  <tabstop>raw_show_saturation</tabstop>
  <tabstop>cartesian_pixel_size</tabstop>
  <tabstop>cartesian_plane_normal_rotate_x</tabstop>
  <tabstop>cartesian_virtual_plane_distance</tabstop>
  <tabstop>cartesian_plane_normal_rotate_y</tabstop>
  <tabstop>polar_pixel_size_tth</tabstop>
  <tabstop>polar_pixel_size_eta</tabstop>
  <tabstop>polar_res_tth_min</tabstop>
  <tabstop>polar_res_tth_max</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>polar_apply_snip1d</sender>
   <signal>toggled(bool)</signal>
   <receiver>polar_snip1d_width</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>polar_apply_snip1d</sender>
   <signal>toggled(bool)</signal>
   <receiver>polar_snip1d_numiter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
